certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -160,9 +160,9 @@ def deploy_cert(self, domain, cert_path, key_path,
         stapling_directives = []
         if self.version >= (1, 3, 7):
             stapling_directives = [
-                ['\n', 'ssl_trusted_certificate', ' ', chain_path],
-                ['\n', 'ssl_stapling', ' ', 'on'],
-                ['\n', 'ssl_stapling_verify', ' ', 'on'], ['\n']]
+                ['\n    ', 'ssl_trusted_certificate', ' ', chain_path],
+                ['\n    ', 'ssl_stapling', ' ', 'on'],
+                ['\n    ', 'ssl_stapling_verify', ' ', 'on'], ['\n']]
```

Comment:
Question: Can you explain to me the reason for the change here? I'm assuming it's to indent these directives. If so, a couple follow up questions:

1. Will this work if the block is has more or less indentation than what is seen here?
2. Can the parser not handle this for us when adding new directives?